index.js

```diff
@@ -242,7 +242,7 @@ class Ky {
 			this.request = new globals.Request(new globals.Request(url, this.request), this._options);
 		}
 
-		if (this._options.json !== undefined) {
+		if (this._options.hasOwnProperty('json')) {
```

Comment:
This will make it hard to _unset_ the `json` option. Also, it will try to stringify and send `undefined` which will cause `JSON.parse()` on the server to throw an error.

I think `!== undefined` is the way to go.

Reply:
to unset the prop, you'd `delete` it... the tricky thing here is that setting `json` is (described as) a "shortcut" for setting `.body = JSON.stringify(...)`, so presumably anything you can pass to `JSON.stringify` should be able to be passed to `json`!  and, `undefined` will be stringified to `null` so the server will be required to parse that (which should be fine).

Having the `json` prop act ever so slightly different than how it's described is IMHO a recipe for weird edge cases...

Reply:
> `undefined` will be stringified to `null`

This is incorrect. `JSON.stringify(undefined)` returns `undefined`, which `JSON.parse()` fails to parse.